api/src/main/java/io/opencensus/trace/propagation/TextFormat.java

```diff
@@ -45,7 +45,7 @@
  *   }
  * }
  *
- * void makeHttpRequest() {
+ * void makeHttpRequest(HttpURLConnection connection) {
  *   Span span = tracer.spanBuilder("Sent.MyRequest").startSpan();
  *   try (Scope s = tracer.withSpan(span)) {
  *     HttpURLConnection connection =
```

Comment:
Should this declaration of `connection` be removed now?

Reply:
done. my bad of adding as param.